### PR TITLE
Update readme to address macos error 

### PR DESCRIPTION
**Pull request by mrjones-plip**
per #411

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,6 +24,33 @@ To install, follow the [installation instructions for your Operating System](htt
...
+sudo launchctl limit maxfiles 65535 200000
+ulimit -n 65535
+sudo sysctl -w kern.maxfiles=100000
+sudo sysctl -w kern.maxfilesperproc=65535
+hugo server
+```
+
+Save this in file with the `.commmand` suffix (e.g. `cht-docs-server.command`) to enable an easy double clicking to start the server. 
+
+Note - be sure to enter your MacOS user password in the terminal when prompted.
```
Comment (abbyad) [inline README.md:52]: Any time I see "Note [...]" I think it is worth formatting as a note (or rewording).

Comment (abbyad) [reply README.md:52]: But then I realize sometimes that its is a GH readme, and not for the documentation site!

**Review by abbyad — CHANGES_REQUESTED** [review]
Good to have this included!

I am in favour of easily digestable readmes, and concerned that this new section is given a lot of visual importance. I'd suggest having a note in the `Building the Documentation` section that mentions the error and links to a troubleshooting file with the workaround.

File: README.md (hunk 1 above)
Comment (mrjones-plip) [reply README.md]: ;)

**Comment by mrjones-plip** [comment]
Excellent feedback to have this take up less visual feedback - thanks! I've updated this PR, but you can easily see the in the [branch's readme here](https://github.com/medic/cht-docs/tree/411-document-pipe-failed-macos-error#building-the-documentation).

**Review by abbyad — APPROVED** [review]
Looks great like this!!

**Pull request merged**